classes/services/queryBuilders/SubmissionListQueryBuilder.inc.php

```diff
@@ -44,6 +44,38 @@ public function filterBySections($sectionIds) {
 	 * @return object Query object
 	 */
 	public function appGet($q) {
+		$primaryLocale = \AppLocale::getPrimaryLocale();
+		$locale = \AppLocale::getLocale();
+
+		$this->columns[] = Capsule::raw('COALESCE(stl.setting_value, stpl.setting_value) AS section_title');
```

Comment:
It looks to me like heavy use of `Capsule::raw` essentially reverts us to gluing together SQL strings, negating the usefulness of the query builder, no?

Reply:
I think that's OK to wrap `Capsule::raw` around this example, as I suspect it'll try to interpret it as a string value or something without the wrapper.